rialto_airflow/database.py

```diff
@@ -141,6 +141,7 @@ class Funder(Base):  # type: ignore
     name = Column(String, nullable=False)
     grid_id = Column(String, unique=True)
     ror_id = Column(String, unique=True)
+    openalex_id = Column(String, unique=True)
```

Comment:
Saving the OpenAlexID means we can use it to look up a Funder instead of always needing to go to the API.